<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server</class>
 <widget class="QWidget" name="server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#server {background-image: url(:/new/prefix1/image/1106022.jpg);}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>服务器端</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame_client_on">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>211</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="le_usernum_on">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>在线用户数量：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_on">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_client_off">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>211</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>离线用户数量：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_usernum_off">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_off">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_save">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存用户信息</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
